packages/ui/src/memberships/queries/members.graphql

```diff
@@ -140,4 +143,7 @@ query GetMemberRowDetails($workerId_in: [ID!]!) {
   terminatedWorkerEventsConnection(where: { worker: { id_in: $workerId_in } }) {
     totalCount
   }
+  memberInvitedEventsConnection(where: { invitingMember: { id_in: $workerId_in } }) {
+    totalCount
+  }
```

Comment:
This invited count is only used on the my member page and I'm not sure of the cost of an extra event type to query, so I think it would be better to add this in a new query/hook which would also query slashed and terminated